src/Computer.js

```diff
@@ -0,0 +1,68 @@
+import { Random } from '@woowacourse/mission-utils';
+
+class Computer {
+  constructor() {
+    this.answer = this.randomNumGenerator();
+  }
+
+  /**
+   *
+   * @returns {[number, number, number]}
+   */
+  randomNumGenerator() {
+    const numbers = [];
+    while (numbers.length < 3) {
+      const number = Random.pickNumberInRange(1, 9);
+
+      if (!numbers.includes(number)) {
+        numbers.push(number);
+      }
+    }
+
+    return numbers;
+  }
+
+  /**
+   *
+   * @param {[number, number, number]} input
+   * @returns {number}
+   */
+  countStrike(input) {
+    let strike = 0;
+
+    input.forEach((num, idx) => {
+      if (num == this.answer[idx]) strike += 1;
+    });
+
+    return strike;
+  }
+
+  /**
+   *
+   * @param {[number, number, number]} input
+   * @returns {number}
+   */
+  countBall(input) {
+    let ball = 0;
+
+    input.forEach((num, idx) => {
+      if (this.answer.includes(num) && num != this.answer[idx]) ball += 1;
```

Comment:
```suggestion
const isBall = this.answer.includes(num) && num != this.answer[idx];
if (isBall) ball += 1;
```

 
조건이 길면 변수화해서 이름을 지어주는 것도 좋은 방법인 것 같아요!

Reply:
아래 리뷰와 더불어 변수를 활용하라는 피드백 감사합니다! 좋은 방법 알아가네요.! :)